pennylane/pulse/convenience_functions.py

```diff
@@ -21,32 +22,45 @@
 except ImportError:
     has_jax = False
 
-
-def constant(windows: List[Tuple[float]] = None):
-    """Returns a callable ``f(p, t)`` that returns ``p`` inside the time
-    windows defined in ``windows``.
+# pylint: disable=unused-argument
+def constant(scalar, time):
+    """Parametrized function that is constant in time and returns the given ``scalar``.
 
     Args:
-        windows (Tuple[float, Tuple[float]]): List of tuples containing time windows where
-        ``f(p, t)`` is evaluated. If ``None``, it is always evaluated. Defaults to ``None``.
+        scalar (float): the scalar to be returned
+        time (float): Time. This argument is not used.
+
+    This function is mainly used to build a parametrized hamiltonian that can be differentiated
+    with respect to its constant term.
```

Comment:
In order to make clear that constant does not mean untrainable, which would be confusing :)
```suggestion
    This function is mainly used to build a parametrized Hamiltonian that can be differentiated
    with respect to its time-independent term. It is an alias for `lambda scalar, t: scalar`.
```